Tests/DataPipeline/DataPipelineInteractionTests.swift

```diff
@@ -175,9 +159,6 @@ class CustomerIOImplementationTest: UnitTest {
 
         customerIO.clearIdentify()
 
-        XCTAssertEqual(hooksMock.profileIdentifyHooksGetCallsCount, 1)
-        XCTAssertEqual(profileIdentifyHookMock.beforeProfileStoppedBeingIdentifiedCallsCount, 1)
-
         XCTAssertNil(profileStoreMock.identifier)
```

Comment:
Would you want to consider updating the method name since we are no longer making use of hooks here.

Reply:
See my [reply above](https://github.com/customerio/customerio-ios/pull/473#discussion_r1448462344)